bcda/cclf/cclf.go

```diff
@@ -275,7 +277,24 @@ func importCCLF(ctx context.Context, fileMetadata *cclfFileMetadata, importer im
 			continue
 		}
 
-		err = importer.do(ctx, txn, cclfFile.ID, b)
+		const (
+			mbiStart, mbiEnd   = 0, 11
+			hicnStart, hicnEnd = 11, 22
+		)
+		cclfBeneficiary := models.CCLFBeneficiary{
```

Comment:
If we're parsing the data with assumption that it's a CCLF8 file, we should remove `importCCLF` and push all of this code into `importCCLF8`.

Reply:
Bump on this, I don't think we need the `importCCLF` function. `importCCLF` used to be generic, but now that parse the file and assume it's a CCLF8 file, we should remove `importCCLF` and move all of the logic into `importCCLF8`.